integration-tests/messages/ipfs.ts

```diff
@@ -0,0 +1,21 @@
+import { unixfs } from '@helia/unixfs';
+import { createHelia } from 'helia';
```

Comment:
js-ipfs is deprecated. Helia is now the standard.

Reply:
did you figure out helia? would be great if we can get a refresher

Reply:
Mostly just used https://github.com/ipfs/helia/wiki/Migrating-from-js-IPFS and the other Helia examples.